### DISCUSSION TITLE
"Filemin" module adds a function to the "Tools" menu for calculating the size of subdirectories

### DISCUSSION THREAD

**Discussion by b77av** [webmin/webmin#2063] [Ideas]
Good afternoon!

Is it possible to add a subdirectory size calculation function to the tools menu?

As an example "du -sh /var/*"

**Comment by iliaross** [comment]
Hello,

If you select a directory and click `Ctrl+S`, it should show its size. It is also available from right-click and selecting **Properties** sub-menu.

**Comment by b77av** [comment]
This function will show the size of just one directory, as well as the space bar.
I'm telling you to count the space in all catalogs. It is very convenient to use when searching for where the free space has disappeared

An example of using such a function in TotalCommander

**Comment by b77av** [comment]
![20 12 23_16 35 01](https://github.com/webmin/webmin/assets/154432803/eea64e8d-076b-4fe5-b552-2fde2577eba3)

**Comment by iliaross** [comment]
I'm puzzled as to why the following wouldn't work for you?


https://github.com/webmin/webmin/assets/4426533/086c937d-df84-4dbf-ad55-77f97446da52

**Comment by b77av** [comment]
Super, that's what you need!!! Thank you very much!

**Reply by iliaross** [reply]
You're welcome.
